platforms/evm/__tests__/integration/tokenBridge.test.ts

```diff
@@ -82,7 +81,7 @@ afterEach(async () => {
   nockBack.setMode('wild');
 });
 
-const network = DEFAULT_NETWORK;
```

Comment:
These tests were actually failing when run locally since the environment would default to TESTNET. The mocked response data is for Mainnet, so this should be set to Mainnet...